client/src/app/modules/components/sections/section-delete/section-delete.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Feel free to delete these component specs

I am not yet sure how the best way to test this app will be, but I'm leaning towards something like cypress instead of the default spec files